examples/amp-story/shopping/remote.json

```diff
@@ -2,15 +2,18 @@
   "items": [
     {
       "product-tag-id": "city-pop",
-      "product-title": "Plastic Love"
+      "product-title": "Plastic Love",
```

Comment:
You'll need to update the `'does use remote config when src attribute is provided'` test in `test-amp-story-shopping-config.js` because it's still expecting the old version of this JSON